README.md

```diff
@@ -247,6 +248,52 @@ Note that:
 - Either `url`, `getUrl`, or `urlPath` must be present.
 - `html` type uses `iframe` and can display `.pdf` if supported by browser.
 
+### <a name="shortcuts"></a>Shortcuts
+
+Configuration for the shortcuts in the document based on focused fields.
+
+```
+{
+ shortcut-name: {
```

Comment:
@harunurhan I tried to use the helper function in the `environment.ts` file but from protractor you cannot access the window object so it gives an error. By executing a script with `browser.executescript()` function you can inject a script in the current open browser of the test but in that case i cannot import my file. So, I just use the this approach to obtain the information before every test scenario from the window object. You can check it in my latest commit here:
https://github.com/inveniosoftware-contrib/ng2-json-editor/pull/100/files#diff-5a31329b1c08e4b38445f4db7d5b13aeR18. Any thoughts?